CarbonQueryDatabase_Adapter/Convert/Foot.cs

```diff
@@ -0,0 +1,54 @@
+/*
+ * This file is part of the Buildings and Habitats object Model (BHoM)
+ * Copyright (c) 2015 - 2020, the respective contributors. All rights reserved.
+ *
+ * Each contributor holds copyright over their respective contributions.
+ * The project versioning (Git) records all such contribution source information.
+ *                                           
+ *                                                                              
+ * The BHoM is free software: you can redistribute it and/or modify         
+ * it under the terms of the GNU Lesser General Public License as published by  
+ * the Free Software Foundation, either version 3.0 of the License, or          
+ * (at your option) any later version.                                          
+ *                                                                              
+ * The BHoM is distributed in the hope that it will be useful,              
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of               
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the                 
+ * GNU Lesser General Public License for more details.                          
+ *                                                                            
+ * You should have received a copy of the GNU Lesser General Public License     
+ * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
+ */
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+using System.ComponentModel;
+using BH.oM.Reflection.Attributes;
+
+namespace BH.Adapter.CarbonQueryDatabase
+{
+    // replace this convert method with central unit conversion methods. 
+    public static partial class Convert
+    {
+        [Description("Convert foot into SI units (meter)")]
```

Comment:
I'm not quite sure if this should be `feet` rather than `foot`. But I think I'll accept what's here as I respect it is a temporary addition while waiting for `Localisation_Toolkit` to wrap `Units.Net` given there is nothing else wrong with this PR.

Reply:
Should be “feet”

Reply:
👍